--- stfl-0.19/Makefile.orig	2008-07-25 13:10:35.000000000 +0200
+++ stfl-0.19/Makefile	2008-07-25 13:12:07.000000000 +0200
@@ -24,6 +24,8 @@
 export CFLAGS += -I. -Wall -Os -ggdb -D_GNU_SOURCE -fPIC
 export LDLIBS += -lncursesw
 
+libdir=lib
+
 all: libstfl.a example
 
 example: libstfl.a example.o
@@ -50,9 +52,9 @@
 	mv -f Makefile.deps_new Makefile.deps
 
 install: all
-	mkdir -p $(DESTDIR)$(prefix)/lib
+	mkdir -p $(DESTDIR)$(prefix)/$(libdir)
 	mkdir -p $(DESTDIR)$(prefix)/include
-	install -m 644 libstfl.a $(DESTDIR)$(prefix)/lib/
+	install -m 644 libstfl.a $(DESTDIR)$(prefix)/$(libdir)
 	install -m 644 stfl.h $(DESTDIR)$(prefix)/include/
 
 ifeq ($(FOUND_SPL),1)
